siad/network.go

```diff
@@ -65,7 +65,10 @@ func (d *daemon) initializeNetwork(addr string, nobootstrap bool) (err error) {
 		// Every 2 minutes, call CatchUp() on a random peer. This helps with
 		// synchronization.
 		for {
-			go d.CatchUp(d.network.RandomPeer())
+			peer, err := d.network.RandomPeer()
+			if err == nil {
```

Comment:
it's weird that you can't just call `for time.Sleep(time.Minute * 2) {`, but yeah this looks fine to me.